server/services/caseloadService.test.ts

```diff
@@ -439,451 +439,484 @@ describe('Caseload Service', () => {
+        },
+      ])
+    })
   })
 
-  it('should sort offenders by conditional release date ascending', async () => {
-    communityService.getManagedOffenders.mockResolvedValue([
-      {
-        nomsNumber: '1',
-        currentOm: true,
-      },
-      {
-        nomsNumber: '2',
-        currentOm: true,
-      },
-      {
-        nomsNumber: '3',
-        currentOm: true,
-      },
-      {
-        nomsNumber: '4',
-        currentOm: true,
-      },
-    ] as CommunityApiManagedOffender[])
-    prisonerService.searchPrisonersByNomisIds.mockResolvedValue([
-      {
-        prisonerNumber: '1',
-        bookingId: '1',
-        status: 'ACTIVE',
-        indeterminateSentence: false,
-        conditionalReleaseDate: '2022-04-30',
-      },
-      {
-        prisonerNumber: '2',
-        bookingId: '2',
-        status: 'ACTIVE',
-        indeterminateSentence: false,
-        conditionalReleaseDate: '2024-04-30',
-      },
-      {
-        prisonerNumber: '3',
-        bookingId: '3',
-        status: 'ACTIVE',
-        indeterminateSentence: false,
-        conditionalReleaseDate: '2021-04-30',
-      },
-      {
-        prisonerNumber: '4',
-        bookingId: '4',
-        status: 'ACTIVE',
-        indeterminateSentence: false,
-        conditionalReleaseDate: '2023-04-30',
-      },
-    ] as Prisoner[])
-
-    prisonerService.getHdcStatuses.mockResolvedValue([
-      new HdcStatus('1'),
-      new HdcStatus('2'),
-      new HdcStatus('3'),
-      new HdcStatus('4'),
-    ])
-
-    const caseload = await caseloadService.getStaffCaseload(user)
-
-    expect(caseload).toStrictEqual([
-      {
-        nomsNumber: '3',
-        bookingId: '3',
-        prisonerNumber: '3',
-        currentOm: true,
-        status: 'ACTIVE',
-        indeterminateSentence: false,
-        conditionalReleaseDate: '2021-04-30',
-        licenceStatus: LicenceStatus.NOT_STARTED,
-        licenceType: LicenceType.AP,
-      },
-      {
-        nomsNumber: '1',
-        bookingId: '1',
-        prisonerNumber: '1',
-        currentOm: true,
-        status: 'ACTIVE',
-        indeterminateSentence: false,
-        conditionalReleaseDate: '2022-04-30',
-        licenceStatus: LicenceStatus.NOT_STARTED,
-        licenceType: LicenceType.AP,
-      },
-      {
-        nomsNumber: '4',
-        bookingId: '4',
-        prisonerNumber: '4',
-        currentOm: true,
-        status: 'ACTIVE',
-        indeterminateSentence: false,
-        conditionalReleaseDate: '2023-04-30',
-        licenceStatus: LicenceStatus.NOT_STARTED,
-        licenceType: LicenceType.AP,
-      },
-      {
-        nomsNumber: '2',
-        bookingId: '2',
-        prisonerNumber: '2',
-        currentOm: true,
-        status: 'ACTIVE',
-        indeterminateSentence: false,
-        conditionalReleaseDate: '2024-04-30',
-        licenceStatus: LicenceStatus.NOT_STARTED,
-        licenceType: LicenceType.AP,
-      },
-    ])
+  describe('getVaryCaseload', () => {
```

Comment:
A lot of this file was just indented into the appropriate `describe` parent... from here down is what is relevant.